src/one_dragon_qt/widgets/multi_selection_combo_box.py

```
@@ -0,0 +1,548 @@
+        # 计算标签总宽度 = 左边距(8) + 文本宽度 + 文本与按钮间距(6) + 关闭按钮(14) + 右边距(4)
+        total_width = 8 + text_width + 6 + 14 + 4
+
+        return total_width
+
+
+class FluentDropdownButton(QFrame):
+    """Fluent 风格下拉按钮，内部显示标签（自动换行）"""
+
+    clicked = Signal()  # 当按钮被点击时发出信号
+
+    def __init__(self, placeholder_text: str = "请选择...", max_width: int = None, parent=None) -> None:
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)